darwin/Classes/SwiftAmplitudeFlutterPlugin.swift

```diff
@@ -67,6 +67,12 @@ import AmplitudeSwift
                 amplitude?.logger?.warn(message: "\(call.method) called but failed.")
             }
 
+        case "getUserId":
+            let userId = amplitude?.getUserId()
+            amplitude?.logger?.debug(message: "Get user Id: \(String(describing: userId))")
```

Comment:
nit: keep aligned with the android plugin
```suggestion
            amplitude?.logger?.debug(message: "Get userId: \(String(describing: userId))")
```